src/main/java/no/nav/veilarbvedtaksstotte/client/dokdistfordeling/DistribuerJournalpostResponsDTO.kt

```diff
@@ -1,3 +1,3 @@
 package no.nav.veilarbvedtaksstotte.client.dokdistfordeling
 
-data class DistribuerJournalpostResponsDTO(val bestillingsId: String)
+data class DistribuerJournalpostResponsDTO(val bestillingsId: String?)
```

Comment:
du er ikke lenger sikker på at vi får inn en id?

Reply:
Det er for å håndtere mulige varianter av 2xx respons med uventet innhold i body, for ikke å forsøke på nytt. https://github.com/navikt/veilarbvedtaksstotte/pull/246/files#diff-7c1bb110ac2cfd69ee85a2dba7fa149b891a87cd1e909a6f3b1b11c997855c35R169